meta-ostro/conf/distro/include/ostroproject-ci.inc

```diff
@@ -31,27 +31,86 @@ SDKMACHINE ?= "x86_64"
 # Generate source mirror tarballs:
 BB_GENERATE_MIRROR_TARBALLS = "1"
 
-# Include the devel settings
+# If (and only if) building in the CI, the BUILD_ID is guaranteed to
+# be consistent across different builds and therefore it can and
+# should be included in the os-release package.
+#
+# When doing the same outside of the CI, we'd end up with an
+# os-release file that has BUILD_ID = ${DATETIME} for the first time
+# that the os-release file was generated. It does not get re-generated
+# (explicit vardepsexclude in os-release.bb) because that would cause
+# constant rebuilds.
+OS_RELEASE_FIELDS_append = " BUILD_ID"
```

Comment:
OS_RELEASE_FIELDS will be used by os-release.bb to put that value also into /etc/os-release.